test/e2e/notifier/create/create.go

```diff
@@ -129,14 +129,7 @@ func (c *context) testCreateResource(t *testing.T) {
 			}
 
 			resource := utils.GVRToString(test.GVR)
-			isAllowed := utils.AllowedEventKindsMap[utils.EventKind{
-				Resource:  resource,
-				Namespace: "all",
-				EventType: config.CreateEvent}] ||
-				utils.AllowedEventKindsMap[utils.EventKind{
-					Resource:  resource,
-					Namespace: test.Namespace,
-					EventType: config.CreateEvent}]
+			isAllowed := utils.CheckOperationAllowed(utils.AllowedEventKindsMap, "all", resource, config.CreateEvent) || utils.CheckOperationAllowed(utils.AllowedEventKindsMap, test.Namespace, resource, config.CreateEvent)
```

Comment:
We can combine move check for "all" namespace in `CheckOperationAllowed`. So for given event, resource and ns, `CheckOperationAllowed` should return true if event is allowed for "all" namespaces.

Reply:
In that case we need to keep the '&&' or '||' opertion as a parameter, since the testcases require OR operation while the controller require AND operation.

Reply:
Chnaged the implementation to have a combination of "all namespace" in `CheckOperationAllowed`